site/content/en/docs/Installation/Terraform/aks.md

```diff
@@ -58,6 +58,9 @@ Configurable parameters:
 - gameserver_minPort - the lower bound of the port range which gameservers will listen on (default is "7000")
 - gameserver_maxPort - the upper bound of the port range which gameservers will listen on (default is "8000")
 - gameserver_namespaces - a list of namespaces which will be used to run gameservers (default is `["default"]`). For example `["default", "xbox-gameservers", "mobile-gameservers"]`
+{{% feature expiryVersion="1.10.0" %}}
```

Comment:
I am totally sorry - this should be "publishVersion" not "expiryVersion".

This is telling the system that it should not be shown, not that it _should_ be shown :facepalm:.

I totally told you the wrong thing.

Reply:
switched, with a nice typo in the commit message :)